### PR TITLE
* Added new function Sum returning an integer sum of two integers.

### PR DESCRIPTION
**Pull request by Ghazigg22**

### CODE REVIEW

File: test_callback.c [hunk 1]
```diff
@@ -23,4 +23,15 @@ int main()
...
 
+}
+
+/**
+  * @brief Sum of two integers function
+  * @param This parameter is an integer
+  * @param This parameter is an integer
+  * @retval This function returns an integer
+  */
+int sum ( int a , intb)
```
Comment (nechi-israa) [inline test_callback.c]: plz chnage it to int b

File: test_callback.c [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #include<stdio.h>
 
 /* Add fonction */ 
-
+int sum ( int a , intb);
```
Comment (nechi-israa) [inline test_callback.c]: plz chnage it to int b

**Review by nechi-israa — CHANGES_REQUESTED** [review]

File: test_callback.c (hunk 2 above)
Comment (Ghazigg22) [reply test_callback.c]: Relevant! Just a typo!
Thanks!

Comment (Ghazigg22) [reply test_callback.c]: Done!

File: test_callback.c (hunk 1 above)
Comment (Ghazigg22) [reply test_callback.c]: Relevant! Just a typo!
Thanks!

Comment (Ghazigg22) [reply test_callback.c]: Done!

**Review by nechi-israa — APPROVED** [review]
Good!

**Pull request merged**